src/sql/workbench/parts/query/browser/queryActions.ts

```diff
@@ -734,6 +752,11 @@ export class ListDatabasesActionItem implements IActionViewItem {
 			return;
 		}
 
+		if (!this._editor.input) {
+			console.error('editor input was null');
+			return undefined;
```

Comment:
Just return - this method doesn't have a return type.

Reply:
You're still returning undefined here - this should just be return;

---
In reply to: [322376405](https://github.com/microsoft/azuredatastudio/pull/7084#discussion_r322376405) [](ancestors = 322376405)